layouts/main.tsx

```diff
@@ -93,6 +95,7 @@ const Layout = ({ children, title = "Livepeer Explorer" }) => {
   const { asPath } = useRouter();
   const { data: protocolData } = useProtocolQuery();
   const { data: pollData } = usePollsQuery();
+  const { data: proposalsData } = useTreasuryProposalsQuery({ pollInterval });
```

Comment:
I'd say we don't even need a poll interval here not to make the website more expensive even if not in the treasury tab. Can just fetch once on page load.

Reply:
Updated! I'll push the commit shortly. I retained `pollInterval` for consistency with `treasury/index.tsx`, though I haven't explored its function in detail.

https://github.com/livepeer/explorer/blob/a16b5343c7209bbe584261dd6b0b242b1c676990/pages/treasury/index.tsx#L20